doc/source/index.rst

```diff
@@ -1,8 +1,16 @@
 PyAEDT documentation  |version|
 ===============================
 
-PyAEDT is a Python library that interacts directly with the API for
-Ansys Electronics Desktop (AEDT) to make scripting simpler.
+**Useful links**:
+`Installation <https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html>`_ |
+`Source Repository <https://github.com/ansys/pyaedt>`_ |
+`Issues <https://github.com/ansys/pyaedt/issues>`_
+
+PyAEDT is a Python library that interacts directly with the Ansys Electronics Desktop (AEDT) API,
+enabling straightforward and efficient automation in your workflow.
+
+In the upper right corner of the documentation’s title bar, you will find a version switcher dropdown.
```

Comment:
This line and the next contain jargon. Rewrite more simply--and avoid future tense: The upper right corner of the documentation’s title bar displays an option for switching between different versions of the documentation.